app/uk/gov/hmrc/emcstfe/connectors/ChrisConnector.scala

```diff
@@ -35,21 +35,28 @@ class ChrisConnector @Inject()(val http: HttpClient,
                                soapUtils: XmlUtils
                               ) extends BaseConnector {
 
-  def postChrisSOAPRequestAndExtractToModel[A](request: ChrisRequest)
+  def postChrisSOAPRequestAndExtractToModel[A](request: ChrisRequest, featureSwitch: Boolean)
```

Comment:
You don't need to pass the feature switch in here, you can just feature switch the url in app config